cmd/plume/release.go

```diff
@@ -346,6 +373,10 @@ func doGCE(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 			plog.Fatal(err)
 		}
 	}
+
+	if err := src.Delete(ctx, src.Prefix() + spec.GCE.Image); err != nil {
+		plog.Fatalf("deleting GCE image from GCS: %v", err)
```

Comment:
Thinking about it, maybe we have to keep it around if direct deletion is not allowed. If we are able to remove the image once published in the marketplace, we could do so in a GC job, which could also delete the GCS object then.

Reply:
Something seems to be implemented directly here, worth checking if it deletes image plus object:
```
18:25:46  2022-09-21T16:25:46Z plume: Deprecating old image flatcar-alpha-3346-0-0
18:25:47  2022-09-21T16:25:46Z plume: Pruning 1 GCE images.
18:25:47  2022-09-21T16:25:46Z plume: Deleting old image flatcar-alpha-3127-0-0
```

Reply:
It runs `api.DeleteImage(old.Name)` but it doesn't delete the object yet.

Reply:
This is in `329:		plog.Noticef("Pruning %d GCE images.", len(oldImages)-spec.GCE.Limit)`

Reply:
Ok, so we need to get the image, access the image raw disk and get the storage link to delete the object

Reply:
Or we could use the calculated GCS storage object that we would have uploaded

Reply:
Not sure because the calculated object is based on the current release - for the image `flatcar-alpha-3127-0-0` it would be a different one.

Reply:
Yes, just using the same scheme for the path with the right version, but your idea is cleaner of course. We could also do a manual cleanup of old GCS objects at some point.